packages/engine/Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -185,6 +194,72 @@ function ArcGisMapServerImageryProvider(options) {
   const resource = Resource.createIfNeeded(options.url);
   resource.appendForwardSlash();
 
+  let accessToken;
+  switch (options.basemap_id) {
+    case "satellite":
```

Comment:
We use enums to manage a defined set of types like this in order to reduce the chance of error (see `ArcType` for an example):

```
/**
 * Types of ArcGIS Basemap layers supported by default.
 *
 * @enum {number}
 * @see ArcGisMapServerImageryProvider
 */
const ArcGisBaseMapType = {
   SATELLITE: 1,
   OCEANS: 2,
   HILLSIDE: 3
}
```